src/modules/line_break.rs

```diff
@@ -1,9 +1,8 @@
 use super::Segment;
-use clap::ArgMatches;
-use std::path::Path;
+use crate::context::Context;
 
 /// Creates a segment for the line break
-pub fn segment(_current_dir: &Path, _args: &ArgMatches) -> Option<Segment> {
+pub fn segment(_context: &Context) -> Option<Segment> {
```

Comment:
What do you think of removing `Context` struct as a argument here? It will make it easier to refactor in the future if we can easily see what does and does not depend on `Context`?

Reply:
I was thinking that the `_` prefix would denote that the argument is unused.